trait2d/analysis/__init__.py

```diff
@@ -1049,26 +1054,35 @@ def normalized(self, normalize_t = True, normalize_xy = True):
         return NormalizedTrack(x, y, t, xmin, ymin, tmin, id=self._id)
 
     def calculate_msd(self):
-        """Calculates the mean squared displacement of the track. Result is stored in the Track object.
+        
+        """Calculates the track's mean squared displacement (msd) and stores it in 'track' object. Also deletes temporary values and colelcts them.
+           Furthermore, calculates the standard deviation per point of the MSD array (msd_error) and the standard error of the mean (SEM) [legacy version].
         """
+        
+        import gc
```

Comment:
You shouldn't need to import that (see below).

Reply:
Actually, I think you need to import it to use it manually.
However, with the comments above it is obvious that using the GC here is obsolete.